fusequery/query/src/servers/server.rs

```diff
@@ -50,10 +50,10 @@ impl ShutdownHandle {
         let sessions = self.sessions.clone();
         let join_all = futures::future::join_all(shutdown_jobs);
         async move {
-            if !shutdown.load(Ordering::Relaxed) {
+            if !shutdown.load(Ordering::Acquire) {
```

Comment:
https://doc.rust-lang.org/std/sync/atomic/enum.Ordering.html#variant.Acquire

Acquire only affects instruction reorder(in single thread). It seems that this is also thread unsafe?

Reply:
sorry about this,  change it to cas simplely